src/ExperimentalStableMemory.mo

```diff
@@ -59,7 +62,10 @@ module {
   /// let afterSize = StableMemory.size();
   /// afterSize - beforeSize // => 10
   /// ```
-  public let size : () -> (pages : Nat64) = Prim.stableMemorySize;
+  public func size(): (pages : Nat64) {
+    Prim.debugPrint(deprecationWarning);
+    Prim.stableMemorySize();
+  };
```

Comment:
```suggestion
  /// @deprecated Please use the StableRegions library instead: https://internetcomputer.org/docs/current/motoko/main/stable-memory/stable-regions/#the-region-library
  public let size : () -> (pages : Nat64) = Prim.stableMemorySize;
```
I believe this would be the canonical way of deprecation: With the annotation, the compiler emits the deprecation warning, whereas otherwise, the warning would be emitted (and in future logged) for each call at runtime.